src/securejoin/securejoin_tests.rs

```diff
@@ -243,7 +243,7 @@ async fn test_setup_contact_ex(case: SetupContactCase) {
         .unwrap();
     match case {
         SetupContactCase::AliceHasName => assert_eq!(contact_alice.get_authname(), "Alice"),
-        _ => assert_eq!(contact_alice.get_authname(), "Alice Exampleorg"),
+        _ => assert_eq!(contact_alice.get_authname(), ""),
```

Comment:
The test here was wrong before: It asserted that Bob knows Alice's authname, even though the commend a few lines above says `// Make sure Alice hasn't yet sent their name to Bob.`. This is now fixed, because `get_chat()` doesn't transfer the name anymore.